packages/cli/tests/e2e/debug/DebugNonSsoTab.tests.ts

```diff
@@ -25,7 +23,7 @@ import {
 } from "../commonUtils";
 import { deleteAadAppByObjectId, deleteTeamsApp, getTeamsApp } from "./utility";
 
-describe("Debug V3 sso-tab template", () => {
```

Comment:
Are sso-related actions still tested in other template? M365?

Reply:
Yes, in DebugM365Tab.